project_app/pages/createreview.tsx

```diff
@@ -36,12 +59,44 @@ export default function Reviews() {
 
         // Name of game on load
         setSelectedGame(games[0].id);
+        setReviewValues((prevState) => ({
+            ...prevState,
+            game: games[0].id
+          }));
+        setStratValues((prevState) => ({
+            ...prevState,
+            game: games[0].id
+          }));
     }
       
     useEffect(() => {
         fetchGames();
     }, []);
 
+    function handleInputChange(e) {
+        const value = e.target.value;
+        const id = e.target.id;
+        if (reviewOrStrat === 'Review') {
+            setReviewValues((prevState) => ({
+                ...prevState,
+                [id]: value
+            }));
+            setActiveValues((prevState) => ({
+                ...prevState,
+                [id]: value
+            }));

```

Comment:
can we have this as a boolean instead?

`isReview = true or false`

then just toggle it like `isReview ? review?.map : strat?.map`